frontend/app/routes/home.tsx

```
@@ -0,0 +1,13 @@
+import type { Route } from "./+types/home";
+import { Welcome } from "../pages/welcome/welcome";
+
+export function meta({}: Route.MetaArgs) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)